engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1577,8 +1577,9 @@ private void advanceStop(final VMInstanceVO vm, final boolean cleanUpEvenIfUnabl
         }
 
         try {
-            _networkMgr.release(profile, cleanUpEvenIfUnableToStop);
-            s_logger.debug("Successfully released network resources for the vm " + vm);
+            s_logger.debug("Not releasing network resources until expunge command is sent");
+            //_networkMgr.release(profile, cleanUpEvenIfUnableToStop);
+            //s_logger.debug("Successfully released network resources for the vm " + vm);
```

Comment:
Please remove commented code.  It is cruft, and git history allows us to see how code has changed overtime.